discogs_client/tests/test_all.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import unittest
+import glob
+import discogs_client.tests
+from discogs_client.tests import DiscogsClientTestCase
+
+
+class TestAllCases(DiscogsClientTestCase):
+    def test_all(self):
+        from discogs_client.tests import test_core, test_models, test_fetchers, test_utils
+        suite = unittest.TestSuite(test_core.suite())
+        unittest.TextTestRunner().run(suite) 
+        suite = unittest.TestSuite(test_models.suite())
+        unittest.TextTestRunner().run(suite) 
+        suite = unittest.TestSuite(test_fetchers.suite())
+        unittest.TextTestRunner().run(suite) 
+        suite = unittest.TestSuite(test_utils.suite())
+        unittest.TextTestRunner().run(suite) 
```

Comment:
Hey,
Thanks for the tip, looks like it's completely unneeded at this point. Just needs the MANIFEST.in that Rodney talked about, just working on rebasing these into one commit, do you recommend squashing them into one commit or just deleting the ones that aren't the most recent valid commit?